src/test/java/com/snowflake/kafka/connector/streaming/iceberg/IcebergIngestionSchemaEvolutionIT.java

```diff
@@ -81,6 +81,50 @@ void shouldEvolveSchemaAndInsertRecords(
     assertRecordsInTable();
   }
 
+  @ParameterizedTest(name = "{0}")
+  @MethodSource("prepareData")
+  @Disabled
+  // Schema evolution for structured types is not yet supported
+  void shouldEvolveSchemaAndInsertRecords_structuredData(
```

Comment:
It's going to evolve. I am gonna need it later.

Reply:
Sure, but please extract the common parts of both tests.

Reply:
I will just save it to my notes. Maybe I shouldn't have add it yet - it confuses reviewer and it's not yet needed. It will evolve anyway.